### PR TITLE
2.0: simpler builds, better ergonomics

### PR DESCRIPTION
**Pull request by chriskrycho**
See #16, on top of which this lands. My plan is to release that as 1.3.0 (with a docs build as well once I merge it), then rebase this on top of `master` and merge it and cut a release with it.

The breaking changes:

- drop libkit I’m favor of plain old build scripts
- use `prepublishOnly` for publishing the type definitions, and publish them in the root of the repository instead of in a folder so that `paths` is no longer necessary
- drops Flow types, since no one appears to be using them and I don't have a good way to maintain them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +224,23 @@ const theAnswer = ok(42);
 const theAnswerValue = unwrapOr(0, theAnswer);
 ```
 
+You can also use TypeScript's "type narrowing" capabilities: if you *check* which variant you are accessing, TypeScript will "narrow" the type to that variant and allow you to access the `value` directly if it is available.
```
Comment (bmakuh) [inline README.md]: huh, wow. I... actually didn't know you could do this

**Comment by jirutka** [comment]
> drops Flow types, since no one appears to be using them and I don't have a good way to maintain them

:-1: Then at least remove "(and Flow)" from the project's description here on GitLab.

**Comment by chriskrycho** [comment]
Blast, i thought I had done that. 😞 Sorry about that.

Note that I’m open to someone maintaining them; I just don’t have any good way to maintain them myself.

**Comment by chriskrycho** [comment]
I’ve removed it from the project description. Sorry again – just missed that when removing them from the README and everywhere else!

**Pull request merged**